scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -176,17 +180,25 @@ declare namespace WebdriverIO {
 
     interface Browser {
         config: Config;
+        options: RemoteOptions;
+
+        /**
+         * add command to `browser` or `element` scope
+         */
         addCommand(
             name: string,
             func: Function,
             attachToElement?: boolean
         ): void;
+
+        /**
+         * overwrite `browser` or `element` command
+         */
         overwriteCommand(
             name: string,
             func: (origCommand: Function, ...args: any[]) => any,
             attachToElement?: boolean
         ): void;
-        options: RemoteOptions;
         // ... browser commands ...
     }
```

Comment:
We should add the type definitions for `addLocatorStrategy` missed these in my PR.

Reply:
Will you be able to raise a PR for that?